common/changes/@uifabric/dashboard/refactor-getFocusStyle_2019-04-08-22-36.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "",
+      "packageName": "@uifabric/dashboard",
```

Comment:
Can you delete this change file? Shouldn't be needed now that the dashboard package is gone.